### PR TITLE
IDE-4473 refactor to ui plugin where it should only be used

### PR DESCRIPTION
**Pull request by gamerson**

### CODE REVIEW

File: tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/action/WatchTaskAction.java [hunk 1]
```diff
@@ -81,7 +81,9 @@ public void run(IAction action) {
 
 			IProject project = (IProject)element;
 
-			String jobName = project.getName() + ":" + LiferayGradleCore.LIFERAY_WATCH;
+			String jobName =
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/action/WatchTaskAction.java]: Hey @simonjhy  @jtydhr88 

This is not a good design here.  There is called 'leaky abstraction'.  You see that we are creating a 'jobName' which means this method in WatchTaskAction "knows" how other classes (WatchJob) stores jobs by name (using the pattern project:watch:suffix) and then later queries against, and then creates a "new watchjob" later.  This should be addressed in a followup PR.

File: tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/LiferayGradleUI.java [hunk 2]
```diff
@@ -39,10 +39,14 @@
  * @author Gregory Amerson
  * @author Simon Jiang
  */
-public class GradleUI extends Plugin {
+public class LiferayGradleUI extends Plugin {
 
 	// The plug-in ID
 
+	public static final String LIFERAY_STANDALONE_WATCH_JOB_SUFFIX = "standalone";
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/LiferayGradleUI.java]: The user should not see the word 'standalone' or 'workspace' at the end of the job names.  They don't understand what those means.  @simonjhy  please send another PR (doesn't have to be for 3.5 release, but by 3.6) where we don't use the 'display' name of a job to determine what type of job it is.  IF we need to differentiate between the two, we should do it using jobFamlies not job names.

**Comment by simonjhy** [comment]
yes, I agree. I will changne it later
